### ISSUE TITLE
Run Action Builds in Parallel

### ISSUE THREAD

**Issue by shiftgeist** [jpochyla/psst#59]
It'd be great if the Debian build gets executed when the windows build fails. Sequential builds could also be interesting.

![image](https://user-images.githubusercontent.com/30648730/120012566-f3925d80-bfdf-11eb-8c03-9d702c285cab.png)
(from [psst/actions/runs/885002673](https://github.com/jpochyla/psst/actions/runs/885002673))

---

Edit:

Found this in the docs: `Example not requiring dependent jobs to be successful`

```yaml
jobs:
  job1:
  job2:
    needs: job1
  job3:
    if: always()
    needs: [job1, job2]
```

`needs` may not apply, but `if: always()` sounds promissing.

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-not-requiring-dependent-jobs-to-be-successful

**Comment by jpochyla** [comment]
I'm not sure how to use this for a `matrix`-triggered jobs :/

**Comment by jpochyla** [comment]
This seems to be the culprit: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

**Comment by jpochyla** [comment]
Should be fixed by 11cec9dda10828b470b3cf1f6da9c0acd6b56369. There's another issue with the `deb` job not running if Windows or MacOS builds fail, not sure how to approach that one.
